store/sql_post_store.go

```diff
@@ -471,6 +473,8 @@ func (s SqlPostStore) Search(teamId string, userId string, terms string, isHasht
 				    ORDER BY CreateAt DESC
 				LIMIT 100`, searchType)
 
+			terms = strings.TrimRight(terms, specialSearchCharCutset)
```

Comment:
why TrimRight?  why not replace with spaces?  what happens with strings like "(hello)corey"  I would assume should become "hello corey"